timesketch/lib/utils.py

```diff
@@ -247,9 +247,18 @@ def read_and_validate_jsonl(file_handle):
     # Fields that must be present in each entry of the JSONL file.
     mandatory_fields = ['message', 'datetime', 'timestamp_desc']
     lineno = 0
+    max_length_line_jsonl = current_app.config.get('MAX_LENGTH_LINE_JSONL')
     for line in file_handle:
         lineno += 1
         try:
+            current_len = len(line)
+            if current_len > max_length_line_jsonl:
+                warning_string = (
+                    'Line too long at line {0}: {1}')
+                logger.warning(warning_string.format(
+                    lineno, current_len))
+                continue
```

Comment:
ok, few things here... sorry for the nitpicking...

a) skip the `warning_string` creation, just do

```python
logger.warning(
    'Line {0:d} too long ({1:d} chars which is over the maximum supported value of {2:d}'.format(
        line_number, current_len, max_length_line_jsonl))
```

b) so here the behavior is to skip the line... another option would be to actually ingest it, but instead of skipping it, reducing the line size.. would mean a bit more logic here... that means we need to put a flag here for instance, then load the thing as a JSON, to get a dict, and then figure out what parameter is the `long` one and then shorten it... maybe we can put a TODO here to add that in, not do that now, just have the current behavior of skipping teh line as it is, but it might be better for a future follow up to have the behavior to log the issue, but at the same time ingest the line with a shortened field(s) to make sure it still fits within the size boundaries...

c) instead of `lineno` can you rename that to a more descriptive `line_number`